src/frontend/citizen-portal/src/assets/i18n/en.json

```diff
@@ -182,7 +182,13 @@
     "surname_corporate_name": "Surname or Corporate Name",
     "violation_ticket_number": "Violation Ticket Number",
     "violation_ticket_number_hint": "A Violation Ticket Number is 2 letters and 8 numbers",
-    "violation_time": "Violation Time"
+    "violation_time": "Violation Time",
+    "enter_ticket_information": "Enter your ticket information",
+    "view_ticket_examples": "View ticket examples",
+    "ticket_not_found": "Ticket not found.",
+    "take_photo_upload_image": "Please take photo or upload image.",
+    "contact": "Contact",
+    "close": "Close"

```

Comment:
Great work. My only comment is to try to keep the language module keys sorted - this attempts to make it easier to lookup and maintain this list of keys in the future.

Reply:
Done